### ISSUE TITLE
Fluid quests can double-pull from stacked fluid containers

### ISSUE THREAD

**Issue by evilmrhenry** [Funwayguy/BetterQuesting#379]
Better Questing 2.0.233, Minecraft 1.7.10, and Extra Utilities 1.2.12.

(Unlike most fluid containers, Extra Utilities drums can stack if they have the same fluid and fill level. I assume other fluid containers that can do this would have this problem as well.) 
Create a quest for 10,000mb of water.
Pick up 2 full water drums, note that they stack.
Hit the submit button in the quest.
You now have 2 water drums with 246,000/256,000mb. The quest took 10,000mb from each, for a total of 20,000mb.
This does not occur if the water drums are not stacked.

**Comment by Funwayguy** [comment]
Sorry to kill this issue early but I don't support this version of Better Questing anymore. Too much has changed to backport it and newer versions of Minecraft use a completely different method of storing fluids.

**Comment by evilmrhenry** [comment]
Does this mean the bug in 2.0.233 where repeatable quests with a choice reward don't work right won't get fixed? I don't really care about this fluid bug if it doesn't apply to later versions, but the choice reward setup is bad.

**Comment by Funwayguy** [comment]
TLDR: Unfortunately yes the bug(s) will remain unfixed.

There are so many other bugs in 2.x.x across multiple versions of Minecraft (including critical formatting issues) that just fixing this one bug is not going to make it the mod as a whole any more stable. I'm aware it's a pretty bad situation but it would be a huge time sink to go back and try and fix older versions or backport unfinished code.

When BQ3 in 1.12.2 reaches a stable enough point that I can safely backport it and not have to worry too greatly about mass internal changes for the considerable future, then I'll reconsider.

Note: I am aware of the 3.3.237 alpha build for 1.10.2 but that's highly experimental and not nearly as stable or feature complete as the 1.12.2 branch.

**Comment by evilmrhenry** [comment]
Is the 1.12.2 branch stable enough for modpack use? I could upgrade to that, though it would be annoying.

**Comment by Funwayguy** [comment]
They are comparatively much more stable but still in constant development. It also has a built in backup in case something goes wrong after an update and you need to roll back.

I should mention that the next few updates planned are critical database upgrades and rewrites so things could get a little rough. I will be fixing any issues that do show up fairly quickly though.
